runtime/lua/vim/uri.lua

```diff
@@ -3,6 +3,8 @@
 -- https://tools.ietf.org/html/rfc2732
 -- https://tools.ietf.org/html/rfc2396
 
+local in_windows = vim.loop.os_uname().sysname == 'Windows'
```

Comment:
"is" is the conventional prefix
```suggestion
local is_windows = vim.loop.os_uname().sysname == 'Windows'
```

Reply:
Aha got it